src/plugins/intel_gpu/src/graph/network.cpp

```diff
@@ -345,6 +345,13 @@ network::network(engine& engine,
                  bool is_internal)
     : network(program::build_program(engine, topo, config, is_internal), config, engine.create_stream(config), is_internal) {}
 
+network::network(engine& engine,
+                 const topology& topo,
+                 const ExecutionConfig& config,
+                 std::shared_ptr<InferenceEngine::CPUStreamsExecutor> task_executor,
+                 bool is_internal)
```

Comment:
Do we need this additional constructor? Can't we merge this to the previous one

Reply:
Fixed it. thanks.

Reply:
I will remove it at next step. thanks.

Reply:
Fixed it.